imager/ImagerProfile/tests.py

```diff
@@ -0,0 +1,93 @@
+from django.test import TestCase
+from django.contrib.auth.models import User
+from imagerprofile.models import ImagerProfile
+import factory
+
+
+class UserFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = User
+        django_get_or_create = ('username', 'password')
+
+    username = factory.Sequence(lambda n: "User%03d" % n)
+    password = 'secret'
+
+
+class ProfileFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = ImagerProfile
+        django_get_or_create = ('user',)
+
+    user = factory.SubFactory(UserFactory)
+
+
+class UserTestCase(TestCase):
+    def setUp(self):
+        """create two generic users with different settings"""
+        User.objects.create(username='john_smith', is_active=False)
+        User.objects.create(username='tom_thompson',
+                            first_name='Tom', is_active=True)
+
+    def test_user_has_username(self):
```

Comment:
This is an odd name for this test.  You're not testing that the user has a username, you're asserting that the user has a profile.